src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -428,7 +428,7 @@ namespace NServiceBus
     [System.ObsoleteAttribute("Please use `config.UseTransport<MsmqTransport>().SubscriptionAuthorizer(Authorize" +
         "r);` instead. Will be removed in version 7.0.0.", true)]
     public interface IAuthorizeSubscriptions { }
-    [System.ObsoleteAttribute(@"IHandleMessages<T> now exposes the IMessageHandlerContext parameter. You can use this to access what used to be available in the IBus interface.  If you're trying to send messages outside IHandleMessages<T> use either the provided IBusSession or create one with the IBusSessionFactory. Will be removed in version 7.0.0.", true)]
+    [System.ObsoleteAttribute(@"IHandleMessages<T> now exposes the IMessageHandlerContext parameter. You can use this to access what used to be available in the IBus interface.  If you're trying to send messages outside IHandleMessages<T> use the provided IMessageSession. Will be removed in version 7.0.0.", true)]
```

Comment:
while that was already in there before your changes, I think `If you're trying to send messages outside IHandleMessages<T> use the provided IMessageSession.` is wrong. This could cause users in extension points (e.g. behaviors) try to use the session. What do you think?